doc/how_to/get_started.rst

```diff
@@ -5,37 +5,37 @@
 How to “get started”
 ====================
 
-In this introductory example, you will see how to use the SpikeInterface
-to perform a full electrophysiology analysis. We will download simulated
+In this introductory example, you will see how to use SpikeInterface to
+perform a full electrophysiology analysis. We will download a simulated
 dataset, and we will then perform some pre-processing, run a spike
 sorting algorithm, post-process the spike sorting output, perform
 curation (manual and automatic), and compare spike sorting results.
 
-.. code:: ipython
+.. code:: ipython3
```

Comment:
I would suggest against this. What systems use python by default nowadays?

Python 2 is way behind now, let's hide this shameful past from the users.

Reply:
This is due to the conversion from .py -> .rst. I fixed this in our old files but every time we regenerate the new one this switches back to ipython3. The pygment lexer that rtd uses doesn't like this designation which is why I switched it but this is an annoying mess. Sam mentioned we should look into tuning the .rst converter to try to fix this...

Reply:
Ah, OK, if it was not an intentional thing then feel free to ignore this @chrishalcrow . What a pain.